CardsOfOpprotunity/CardsOfOpprotunity/Views/BoardGame/BoardGameView.swift

```diff
@@ -9,51 +9,160 @@ import SwiftUI
-            
-            VStack(spacing: 50)  {
-                (viewModel.cardOneImage ?? Image("cardBack"))
-                    .resizable()
-                    .scaledToFit()
-                Spacer()
-                (viewModel.cardTwoImage ?? Image("cardBack"))
-                    .resizable()
-                    .scaledToFit()
-            }
-            .hidden(!viewModel.isShowingHand)
-            
-            VStack {
-                Spacer()
-                LoadingButton(title: viewModel.buttonTitle) {
-                    do {
-                        try await viewModel.revealCards()
-                    } catch {
-                        
+            VStack() {
+                HStack() {
+                    Button(action: {
+                        self.presentationMode.wrappedValue.dismiss()
+                    }) {
+                        HStack {
+                            Image(systemName: "chevron.backward")
+                            Text("Back")
+                                .foregroundColor(.blue)
+                        }
                     }
+                    .padding(.leading, 20)
+                    
+                    Spacer()
+                    
+                    Text("Current Bet:")
+                        .font(.title)
+                        .fontWeight(.bold)
+                        .padding(.trailing, 70)
+                    
+                    Spacer()
+                }
+               
+                Text("\(gameData.enteredBet)")
+                    .font(.title)
+                    .fontWeight(.semibold)
+                    .multilineTextAlignment(.center)
+                
+                VStack(spacing: 0) {
+                    CardSelectionView(selectedCardIndex: $viewModel.firstSelectedCardIndex, isOffsetPositive: true)
+                        .scaledToFit()
+                    
+                    HStack{
+                        Text("Player 1: \(gameData.playerOneName)")
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                            .padding(.leading, 0)
+                            .font(.title2.bold())
+                            .padding()
+                        Text("Score: \(viewModel.playerOneScore)")
+                            .font(.title2)
+                            .fontWeight(.bold)
+                            .padding()
+                    }
+                    
+                    LoadingButton(title: viewModel.buttonTitle) {
+                        do {
+                            try await viewModel.revealCards()
+                            //showingResults = true
+                            viewModel.isShowingHand = true
+                        } catch {
+                            
+                        }
+                    }
+                    .disabled(viewModel.userSelectedCards)
+                    .opacity(viewModel.userSelectedCards ? 0.8 : 1)
```

Comment:
The other button has an opacity of `0.6`. If you go with `0.8` make the other one `0.8` as well so the buttons have a uniform look.

from the other button `.opacity(gameData.isGameSetupReady ? 1 : 0.6)`